server/routerlicious/packages/lambdas-driver/src/kafka-service/runner.ts

```diff
@@ -103,30 +106,52 @@ export class KafkaRunner implements IRunner {
 	/**
 	 * Signals to stop the service
 	 */
-	public async stop(): Promise<void> {
-		if (!this.deferred || this.stopped) {
+	public async stop(caller?: string, uncaughtException?: any): Promise<void> {
+		if (this.stopped) {
 			return;
 		}
 
 		this.stopped = true;
+		try {
+			// Stop listening for new updates
+			await this.consumer.pause();
 
-		// Stop listening for new updates
-		await this.consumer.pause();
+			// Stop the partition manager
```

Comment:
Space / tab issue? Run npm run prettier:fix to see if it fixes it?

Reply:
Hey @yunho-microsoft , this is a valid tab because I moved it inside a try-catch block.

Reply:
Sorry for the confusion. I mean is it highlighted because you use tab instead of spaces? It is better to follow the existing pattern to use tabs or spaces for all indentations. Thanks

Reply:
That makes sense, I checked that the existing pattern is to use tabs (like on line 94 and 96 above).
These are highlighted on these lines because git diff is identifying these lines as modified lines and other lines as new lines.